packages/@react-native-windows/cli/src/runWindows/runWindowsOptions.ts

```diff
@@ -158,7 +151,6 @@ export const runWindowsOptions: CommandOption[] = [
   {
     name: '--info',
     description: 'Dump environment information',
-    default: false,
```

Comment:
Be sure to update the typings above to reflect that these can now be undefined instead of always a raw boolean.

Reply:
These typings come from the community cli and already accept undefined.

Reply:
I mean the typings in the RunWindowsOptions type above, that we define.

Various options are set to either boolean, or boolean | undefined, based on whether a default is declared here.